release-tools/generate-release-candidate.py

```diff
@@ -0,0 +1,201 @@
+#! /usr/bin/python
+"""
+About:
+               Copy the Staging Artifiacts from Snapshots to Release Candidate folder 
+
+Prerequisites:
+               Need python 3.7 or higher
+               AWS credentials for S3
+               sudo yum install python-pip
+               sudo pip install boto3
+               sudo pip install pyyaml
+
+Usage: 
+              ./copy_snapshots_to_release.py $BUILD_NUMBER
+
+"""
+import boto3
+import os
+import sys
+import yaml
+import hashlib
+import requests
+
+s3 = boto3.client('s3')
+
+
+# Download file to the local
+def download_file(filename,url):
+    try:
+        response = requests.get(url)
+        with open("downloads/" + filename, 'wb') as file_name:
+            file_name.write(response.content)
+            return response.status_code
+    except:
+        raise
+
+
+# Upload local file to the release folder
+def upload_plugin(plugin,release_bucket_name,plugin_category,rc_folder_path):
+    try:
+        if plugin_category != "":
```

Comment:
I dont see a particular case where any plugin does not have category, what is the reason for this branching?

Reply:
That condition was initially added when we wanted the ES and Kibana at the root folder instead of upstream. I think it can be removed now.

Reply:
Removed the condition as we are using upstream for the ES and Kibana files.